Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -53,13 +53,12 @@ static int OSDTime;
 
 Renderer *g_renderer = nullptr;
 
+std::atomic<bool> Renderer::s_bScreenshot;
```

Comment:
Please use Common::Flag which is one of our abstractions around atomics (Common/Flag.h).